test/generate.test.ts

```diff
@@ -592,4 +611,64 @@ describe('generate', function () {
       });
     });
   });
+
+  describe('nominalColorScaleForHighCardinality', () => {
```

Comment:
How about wrap these two `describe` under `describe('stylizer')`.

(Keep `describe('nominalColorScaleForHighCardinality', ...)` as nested under `stylizer` though)

And move the generic case above to here.  (It `should ... when all stylizer are disabled`.)